docs2/concepts/backend.qmd

```diff
@@ -1,6 +1,6 @@
 # Backend
 
-A backend is where execution of Ibis table expressions occur after compiling into some intermediate representation. A backend is often a database and the intermediate representation often SQL, but several types of backends exist. See the [backends page](../backends/index.md) for specific documentation on each.
```

Comment:
I removed the bit referencing `backends/index.md`, since our quarto docs don't seem to have an index for that page (which is fine!)